src/components/Header/NavComponents/userprofile.tsx

```diff
@@ -30,6 +41,73 @@ export const UserProfile: React.FC<{}> = () => {
   const { data: agaveBalance } = useAmountAvailableToStake(address);
   const userBal = agaveBalance ? bigNumberToString(agaveBalance, 3) : "0";
 
+  const chainAddresses = useChainAddresses();
+
+  // internalAddressesPerNetworkId cannot be indexed for each chain, strigified and parsed back to index
+  var chains =  JSON.parse(JSON.stringify(internalAddressesPerNetworkId))
+
+  // Buttons to change to every available chain
+  let buttons : any[] = [];
+  Object.entries(internalAddressesPerNetworkId).forEach(([name, chain]) => {
+    buttons.push(
+      <Button 
+        bg={mode({ base: "primary.500", md: "primary.500" }, "primary.500")}
+        colorScheme="teal"
+        size="xl"
+        h="40px"
+        onClick={() => changeId(chain.chainId)}
+      >
+        {chain.chainName}
+      </Button>
+  )})
```

Comment:
Run the `prettier` formatter to add semicolons and fix indentation appropriately.

Reply:
Done